db/views/analytics_enrollments_v02.sql

```diff
@@ -0,0 +1,139 @@
+  "MonthsHomelessThisTime",
+  "StatusDocumented",
+  "HousingStatus",
+  "DateOfEngagement",
+  "InPermanentHousing",
+  "MoveInDate",
+  "DateOfPATHStatus",
+  "ClientEnrolledInPATH",
+  "ReasonNotEnrolled",
+  "WorstHousingSituation",
+  "PercentAMI",
+  "LastPermanentStreet",
+  "LastPermanentCity",
+  "LastPermanentState",
+  "LastPermanentZIP",
+  "AddressDataQuality",
+  "DateOfBCPStatus",
+  "EligibleForRHY",
+  "ReasonNoServices",
+  "SexualOrientation",
+  "FormerWardChildWelfare",
+  "ChildWelfareYears",
+  "ChildWelfareMonths",
+  "FormerWardJuvenileJustice",
+  "JuvenileJusticeYears",
+  "JuvenileJusticeMonths",
+  "HouseholdDynamics",
+  "SexualOrientationGenderIDYouth",
+  "SexualOrientationGenderIDFam",
+  "HousingIssuesYouth",
+  "HousingIssuesFam",
+  "SchoolEducationalIssuesYouth",
+  "SchoolEducationalIssuesFam",
+  "UnemploymentYouth",
+  "UnemploymentFam",
+  "MentalHealthIssuesYouth",
+  "MentalHealthIssuesFam",
+  "HealthIssuesYouth",
+  "HealthIssuesFam",
+  "PhysicalDisabilityYouth",
+  "PhysicalDisabilityFam",
+  "MentalDisabilityYouth",
+  "MentalDisabilityFam",
+  "AbuseAndNeglectYouth",
+  "AbuseAndNeglectFam",
+  "AlcoholDrugAbuseYouth",
+  "AlcoholDrugAbuseFam",
+  "InsufficientIncome",
+  "ActiveMilitaryParent",
+  "IncarceratedParent",
+  "IncarceratedParentStatus",
+  "ReferralSource",
+  "CountOutreachReferralApproaches",
+  "ExchangeForSex",
+  "ExchangeForSexPastThreeMonths",
+  "CountOfExchangeForSex",
+  "AskedOrForcedToExchangeForSex",
+  "AskedOrForcedToExchangeForSexPastThreeMonths",
+  "WorkPlaceViolenceThreats",
+  "WorkplacePromiseDifference",
+  "CoercedToContinueWork",
+  "LaborExploitPastThreeMonths",
+  "HPScreeningScore",
+  "LOSUnderThreshold",
+  "PreviousStreetESSH",
+  "UrgentReferral",
+  "TimeToHousingLoss",
+  "ZeroIncome",
+  "AnnualPercentAMI",
+  "FinancialChange",
+  "HouseholdChange",
+  "EvictionHistory",
+  "SubsidyAtRisk",
+  "LiteralHomelessHistory",
+  "DisabledHoH",
+  "CriminalRecord",
+  "SexOffender",
+  "DependentUnder6",
+  "SingleParent",
+  "HH5Plus",
+  "IraqAfghanistan",
+  "FemVet",
+  "ThresholdScore",
+  "ERVisits",
+  "JailNights",
+  "HospitalNights",
+  "RunawayYouth",
+  "processed_hash",
+  "processed_as",
+  "roi_permission",
+  "last_locality",
+  "last_zipcode",
+  "source_hash",
+  "SexualOrientationOther",
+  "history_generated_on",
+  "original_household_id",
+  "service_history_processing_job_id",
+  "MentalHealthDisorderFam",
+  "AlcoholDrugUseDisorderFam",
+  "ClientLeaseholder",
+  "HOHLeasesholder",
+  "IncarceratedAdult",
+  "PrisonDischarge",
+  "CurrentPregnant",
+  "CoCPrioritized",
+  "TargetScreenReqd",
+  "HOHLeaseholder",
+  "EnrollmentCoC",
+  "RentalSubsidyType",
+  "TranslationNeeded",
+  "PreferredLanguage",
+  "PreferredLanguageDifferent",
+  "VAMCStation",
+  "project_pk",
+  "MentalHealthConsultation",
+  "DateCreated",
+  "DateUpdated",
+  "UserID",
+  "DateDeleted",
+  "ExportID"
+FROM "Enrollment"
+WHERE "DateDeleted" is NULL
```

Comment:
Should we expose "wip" enrollments? I'm guessing it's better to keep those internal
```suggestion
WHERE "DateDeleted" is NULL AND "ProjectID" is not NULL
```